src/compute-client/src/controller.rs

```diff
@@ -554,6 +554,10 @@ impl<T> ActiveComputeController<'_, T> {
     }
 }
 
+/// Default value for `idle_arrangement_merge_effort` if none is supplied.
+// TODO(#16906): Test if 1000 is a good default.
+const DEFAULT_IDLE_ARRANGEMENT_MERGE_EFFORT: u32 = 1000;
```

Comment:
What do you think about making this configurable by `ALTER SYSTEM`  or LaunchDarkly?

Reply:
I'm torn on this! On one hand it might be useful to for general performance tuning, on the other hand changing this value for an environment would mean that all replicas would have to restart to make it take effect, so it is somewhat risky to expose it as a config option. Also, I'd hope that after #16906 we'll never need to touch this value again, at which point configuration plumbing for it would be basically dead code.